apps/backend/src/build/createBuildDiffs.ts

```diff
@@ -23,25 +23,19 @@ const getBuildType = ({
   return "check";
 };
 
-async function getOrCreateBaseScreenshotBucket(
-  build: Build,
-  { trx }: { trx?: TransactionOrKnex | undefined } = {},
-) {
+async function getOrRetrieveBaseScreenshotBucket(build: Build) {
   if (build.baseScreenshotBucket) {
     return build.baseScreenshotBucket;
   }
 
-  const baseScreenshotBucket = await getBaseScreenshotBucket({
-    build,
-    trx,
-  });
+  const baseScreenshotBucket = await getBaseScreenshotBucket(build);
 
   if (baseScreenshotBucket) {
-    await Build.query(trx)
+    await Build.query()
```

Comment:
If you remove the transaction here, the build can be updated several times in case or sharding, but it's ok